Analysis/include/Luau/ConstraintGenerator.h

```diff
@@ -478,6 +479,8 @@ struct ConstraintGenerator
 
     void fillInInferredBindings(const ScopePtr& globalScope, AstStatBlock* block);
 
+    std::pair<std::vector<TypeId>, std::vector<TypePackId>> getExplicitTypeIds(const ScopePtr& scope, const AstArray<AstTypeOrPack>& explicitTypes);
```

Comment:
This seems like a very weird interface for the constraint generator to have, even ignoring the name. What are these types instantiating?